app/views/support_interface/staff/index.html.erb

```diff
@@ -20,6 +20,16 @@
       row.value { staff.created_at.to_s }
     end
 
+    summary_list.row do |row|
+      row.key { "Assessor" }
+      row.value { staff.with_award_decline_permission? ? "Yes" : "No" }
```

Comment:
For being able to visually see this information quickly, I wonder if it's worth using [`govuk_tag`](https://govuk-components.netlify.app/components/tag/) similar to show we do below for accepted and not accepted.